Comanche055/TAGS_FOR_RELATIVE_SETLOC.agc

```diff
@@ -399,11 +399,11 @@ THISAXIS	=	UNITX
 ERASID		EQUALS	LOW10			# DOWNLINK ERASABLE DUMP ID
 DELAYNUM	EQUALS	THREE
 
-#********************************************************************************************************
+#***************************************************************************************************************
 
-# THE FOLLOWING ECADRS ARE DEFINED TO FACILITATE EBANK SWITCHING.  THEY ALSO MAKE IT EASIER FOR
+#   THE FOLLOWING ECADRS ARE DEFINED TO FACILITATE EBANK SWITCHING.  THEY ALSO MAKE IT EASIER FOR
 # ERASABLE CONTROL TO REARRANGE ERASABLE MEMORY WITHOUT DISRUPTING THE PROGRAMS WHICH SET EBANKS.
-# PRIOR TO ROPE RELEASE FIXED MEMORY CAN BE SAVED BY SETTING EACH EBXXXX =EBANKX (X=4,5,6,7).  EBANKX OF COURSE
+# PRIOR TO ROPE RELEASE FIXED MEMORY CAN BE SAVED BY SETTING EACH EBXXXX =EBANKX (X=4.5.6.7).EBANKX OF COURSE
```

Comment:
I am fairly certain these are commas that have been badly printed (quite common). They have a slight bump in the bottom left, compared to the periods that are symmetrical:

![](https://i.imgur.com/PMTqRVd.png)

Better example at the top of the page:

![](https://i.imgur.com/BeRjhvl.png)

Reply:
I see. I also was unsure about the dots but to me it just seemed like this was more or less intentional. Fixed here:

```suggestion
# PRIOR TO ROPE RELEASE FIXED MEMORY CAN BE SAVED BY SETTING EACH EBXXXX =EBANKX (X=4,5,6,7).EBANKX OF COURSE
```

Reply:
I see. I also was unsure about the dots but to me it just seemed like this was more or less intentional. Fixed here:

```suggestion
# PRIOR TO ROPE RELEASE FIXED MEMORY CAN BE SAVED BY SETTING EACH EBXXXX =EBANKX (X=4,5,6,7).EBANKX OF COURSE
```

Reply:
I see. I also was unsure about the dots but to me it just seemed like this was more or less intentional. Fixed here:

```suggestion
# PRIOR TO ROPE RELEASE FIXED MEMORY CAN BE SAVED BY SETTING EACH EBXXXX =EBANKX (X=4,5,6,7).EBANKX OF COURSE
```

Reply:
I see. I also was unsure about the dots but to me it just seemed like this was more or less intentional. Fixed here:

```suggestion
# PRIOR TO ROPE RELEASE FIXED MEMORY CAN BE SAVED BY SETTING EACH EBXXXX =EBANKX (X=4,5,6,7).EBANKX OF COURSE
```

Reply:
I see. I also was unsure about the dots but to me it just seemed like this was more or less intentional. Fixed here:

```suggestion
# PRIOR TO ROPE RELEASE FIXED MEMORY CAN BE SAVED BY SETTING EACH EBXXXX =EBANKX (X=4,5,6,7).EBANKX OF COURSE
```

Reply:
I see. I also was unsure about the dots but to me it just seemed like this was more or less intentional. Fixed here:

```suggestion
# PRIOR TO ROPE RELEASE FIXED MEMORY CAN BE SAVED BY SETTING EACH EBXXXX =EBANKX (X=4,5,6,7).EBANKX OF COURSE
```

Also sorry for my comment turning up several times, seems [Github is not feeling so good right now](https://status.github.com).

Reply:
No problem, I can't apply the changes until they've resolved the issue either 😢